### DISCUSSION TITLE
Is it possible to make it generate text continuously, without stopping?

### DISCUSSION THREAD

**Discussion by gustavotoyota** [oobabooga/text-generation-webui#4733] [Q&A]
I wanted to see it generate a whole text by itself, but it keeps stopping in the middle and I have to press Generate or Continue to make it keep going.

My idea is to only have it stop when I click the Stop button.

**Comment by superskirv** [comment]
I dont think you understand what your goal is. It will do one of two things, repeat nonsense, or diverge into random tangets, which usually lead to repeating nonsense. Though I do wonder what would happen if you had a second LLM, starting from scratch, cleaning up the data after each prompt and then resubmitting it.

Either way I would think the context would get so large it would take minutes for a response, and the response would be garbage.

**Comment by HiroseKoichi** [comment]
Ban EOS token is what you're looking for; you can find it in the generation parameters tab.

**Reply by gustavotoyota** [reply]
This parameter helps. But still sometimes the generation stops in the middle of a sentence. Is there some setting for max generated tokens somewhere?

**Reply by gustavotoyota** [reply]
Just found it:

max_new_tokens (int, optional) — The maximum numbers of tokens to generate, ignoring the number of tokens in the prompt.
